<launch>

  
  <!-- We assume that the images (depth and rgb) are already aligned-->
  <arg name="depth_camera_info_topic" default="/camera/aligned_depth_to_color/camera_info" />
  <arg name="rgb_camera_info_topic" default="/camera/color/camera_info" />
  <arg name="rgb_image_topic" default="/camera/color/image_raw" />
  <arg name="depth_image_topic" default="/camera/aligned_depth_to_color/image_raw" />

  <node name="rgbd_camera" pkg="rgbd_camera" type="rgbd_camera_node.py" args="$(arg depth_camera_info_topic) $(arg rgb_camera_info_topic) $(arg rgb_image_topic) $(arg depth_image_topic)" output="screen"/>

</launch>